test/__snapshots__/index.test.js.snap

```diff
@@ -165,30 +165,30 @@ async-ts
 
 
 Build \\"asyncTs\\" to dist:
-103 B: async-ts.js.gz
-72 B: async-ts.js.br
-112 B: async-ts.esm.js.gz
-91 B: async-ts.esm.js.br
-202 B: async-ts.umd.js.gz
-159 B: async-ts.umd.js.br"
+73 B: async-ts.js.gz
+57 B: async-ts.js.br
+80 B: async-ts.esm.js.gz
+66 B: async-ts.esm.js.br
+186 B: async-ts.umd.js.gz
+143 B: async-ts.umd.js.br"
 `;
 
 exports[`fixtures build async-ts with microbundle 2`] = `7`;
 
 exports[`fixtures build async-ts with microbundle 3`] = `
-"var o=function(){function o(){}return o.prototype.foo=function(){return Promise.resolve()},o}();export{o as MyClass};
+"class e{foo(){return Promise.resolve()}}export{e as MyClass};
```

Comment:
Seems like something is causing us to output modern code in non-modern bundles. Not sure which dep but my guess is `preset-env` or `typescript`?